objects.js

```diff
@@ -8520,6 +8523,48 @@ WatcherMorph.prototype.userMenu = function () {
     return menu;
 };
 
+WatcherMorph.prototype.valueExporter = function (format) {
+    var myself = this,
+        value = this.currentValue,
+        contents,
+        format = format || 'plain',
+        ide = myself.parentThatIsA(IDE_Morph);
+
+    return function () {
+        switch (format) {
+            case 'plain':
+                contents = value instanceof List ? 
+                    value.asArray().join(', ') : 
```

Comment:
Maybe it's an interpretation of a CSV, but I always assumed a 1D CSV would be separated by newlines, because in a 2D CSV, the `,` separates the items of the inner dimension. A CSV file, just like a TSV is both a comma (or tab) _and_ newline separated file. 
In that case, the comma is the field separate and the newline is the record separator. 

If `[ [1, 2, 3], [a, b, c] ]` is turned into a CSV of:
```
1,2,3
a,b,c
```
Then I think it's reasonable to expect a newline to separate items. Each item in a 1D list is a separate record. When I think about how most 1D lists are used, I think this is more common and also maps fairly well to the idea of rows in a relational DB. 

In any event, I really see very list use for a text file that's one row. Newlines make the output data much easier to read, and work with.